.github/workflows/freebsd.yml

```diff
@@ -31,7 +31,12 @@ jobs:
         uses: vmactions/freebsd-vm@v1
         with:
           prepare: |
-            pkg install -y gcc gmake
+            pkg install -y autoconf automake libtool
+            pkg install -y cmake ninja
           run: |
             MAKE=gmake ./configure
             gmake -j2 check || (cat tests/test-suite.log && exit 1)
+            
+            cmake -G Ninja -B build
+            ninja -C build
+            ninja -C build test
```

Comment:
It would be nice to know if only autoconf breaks or only cmake. So could this be split in some kind of matrix?

Reply:
Hmm, splitting it into a matrix would require running two separate VMs, which is probably fine.

Reply:
Yes, I think it should be fine. It seems to run quite quickly.